common/placeholders.js

```diff
@@ -40,6 +41,13 @@ const placeholders = [
     required: true,
```

Comment:
remember to delete this placeholder and the `table` function

Reply:
You do not think it could be use-full later??

Reply:
I don't see how. In any case, if you're not actually planning to use it for something, get rid of it. It'll still be in the git history.